webapp/src/main/webapp/vue-apps/space-creation/components/SpaceCreation.vue

```diff
@@ -45,7 +45,8 @@
 export default {
   methods: {
     addNewSpace() {
-      window.require(['SHARED/spaceForm'], drawer => drawer.open(false, this.$root.availableSpaceTemplates));
+      const spaceTemplates = this.$root.availableSpaceTemplates.filter(t => this.$root.settings.spaceTemplateIds.includes(t.id));
+      window.require(['SHARED/spaceForm'], drawer => drawer.open(false, spaceTemplates));
```

Comment:
`this.$root.spaceTemplates` already references this value

```suggestion
      window.require(['SHARED/spaceForm'], drawer => drawer.open(false, this.$root.spaceTemplates));
```

Reply:
yes, done!